fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/DerbyBootstrapper.java

```diff
@@ -111,12 +111,12 @@ public static void bootstrap(Connection connection, String adminSchemaName, Stri
         // Current version history for the database. This is used by applyWithHistory
         // to determine which updates to apply and to record the new changes as they
         // are applied
-        VersionHistoryService vhs = new VersionHistoryService(adminSchemaName, dataSchemaName);
+        VersionHistoryService vhs = new VersionHistoryService(adminSchemaName, dataSchemaName, OAUTH_SCHEMANAME);
         vhs.setTarget(adapter);
         vhs.init();
 
         // Use the version history service to determine if this table existed before we run `applyWithHistory`
-        boolean newDb = vhs.getVersion(dataSchemaName, DatabaseObjectType.TABLE.name(), "PARAMETER_NAMES") == null;
+        boolean newDb = vhs.getVersion(dataSchemaName, DatabaseObjectType.TABLE.name(), "PARAMETER_NAMES") == 0;
```

Comment:
should it be `newDb = (x == null || x == 0)`?

Reply:
fixed